app/views/declaration/summary/locations_section.scala.html

```diff
@@ -14,67 +14,44 @@
  * limitations under the License.
  *@
 
-@import models.declaration.Locations
+@import forms.declaration.GoodsLocation
+@import models.requests.JourneyRequest
 @import models.viewmodels.HtmlTableRow
+@import services.Countries._
 
-@(locations: Option[Locations])(implicit messages: Messages)
+@(declarationData: ExportsDeclaration)(implicit messages: Messages, journeyRequest: JourneyRequest[AnyContent])
```

Comment:
Similar

Reply:
For every section we will stay with declarationData as argument. We will use implicit one on the `summaryPage`